task-5/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState } from 'react';
+
+function App() {
+
+    var [fuel, setFeul] = useState(0);
+    var [alertText, setAlertTxt] = useState('Processing...');
+
+    useEffect(() => {
+      fetch('https://new.world.com/fleet/121')
+        .then(response => response.json())
+        .then(json => {
+          setFeul(json);
+          console.log(json);
+        })
+    });
+
+    useEffect(() => {
+      if (!fuel) setAlertTxt('Processing...'); else
+      if (fuel.litres > 0) setAlertTxt('Need to buy more fuel'); else
+      setAlertTxt('All is fine');
+    }, [fuel])
+
+    return (
+      <div>
+        <h1 style={alertText == 'Need to buy more fuel'? {color:"red"}:{}}>{alertText}</h1>

```

Comment:
Would be much better to use SCSS here to style the DOM elements, however if we want to stick to styles, I would say to have the styles defined above the App class statically, as they are not dynamic from what I can see. And you can use the style based on the fact (fuel.litres > 10000), not on the outcome (the alert text being set to a specific string).

I would say to better have different elements and render them based on the fact (fuel.litres > 10000) rather than have an all-round HTML element that renders everything itself.